libbeat/reader/readfile/bench_test.go

```diff
@@ -0,0 +1,66 @@
+package readfile
```

Comment:
Not sure if you all have preferences on where/how to include benchmarks. Happy to move/remove this as desired.

Reply:
It's fine where you put it.